pyproject.toml

```diff
@@ -56,6 +56,8 @@ ignore = [
     # - RUF032 - decimal-from-float-literal
     "RUF032",
     "RUF045",
+    # - UP045 - Use `X | None` instead of `Optional[X]`
```

Comment:
Is this supported by 3.9?

Reply:
Thats why I ignore them, otherwise it gives me lots of errors that I should replace `Optional[X]` with `X | None` which obviously doesnt work in py3.9 which we still support.